src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -269,11 +287,12 @@ def add_image_segmentation(nwbfile: NWBFile, metadata: dict) -> NWBFile:
     return nwbfile
 
 
-def add_two_photon_series(
+def add_photon_series(
     imaging: ImagingExtractor,
     nwbfile: NWBFile,
     metadata: dict,
-    two_photon_series_index: int = 0,
```

Comment:
Can you soft-deprecate for now

Reply:
https://github.com/catalystneuro/neuroconv/pull/462/commits/6817990c78c6b109ab49a113e3feac9e7db0f68c

Reply:
Can you 

(a) change the default of `two_photon_series_index` to `None` 

and if `None`

(b) raise warning as you are, and set `photon_series_index = two_photon_series_index`

Reply:
I hope this time I got it right: https://github.com/catalystneuro/neuroconv/pull/462/commits/d7ce2ff9e6ce65af1a4b578ee0501a0ae4577eee
Thank you for the help!

Reply:
Great, thanks!